easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'PHYLIP'
+version = '3.697'
+
+homepage = 'https://evolution.genetics.washington.edu/phylip'
+description = "PHYLIP is a free package of programs for inferring phylogenies."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```

Comment:
If PHYLIP can benefit from the compiler used then this should be `GCC` - so that an `iccifort` equivalent could be made. This change means that the `builddependencies` below can be dropped.